tests/deploy/test_scheduler_agent.py

```diff
@@ -76,7 +76,8 @@ class DummyExecutor(executor.Executor):
     A dummy executor:
         * It doesn't actually do any deploys, but instead keeps track of the actions
           (execute, dryrun, get_facts, etc.) that were requested on it.
-        * It reports a deploy as failed if the resource has an attribute with the value of the `FAIL_DEPLOY` variable.
+        * It reports a deploy as failed if the resource has an attribute with the value of the `FAIL_COUNT` variable with
```

Comment:
I like the concept, but I think it should be at the resource level rather than at the executor level. If that proves difficult I'd rather revert to the old behavior.

Reply:
Reverted to the old behaviour and tried a workaround. What do you think?

Reply:
Looks OK to me, with one caveat (see comment there). Though I wonder why you need it tbh. You release a new version anyway, so couldn't you simply set the `r1_fail` flag?